packages/design-toolkit/src/components/combobox-field/styles.module.css

```
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+@reference '../../index.css';
+
+@layer components.l1 {
+  .field {
+    @apply flex flex-col gap-xs;
+  }
+
+  .control {
+    @apply flex items-center rounded-medium px-s py-xs outline fg-primary-bold outline-interactive;
+
+    @variant placeholder {
+      @apply fg-primary-muted;
+    }
+
+    @variant hover {
+      @apply outline-interactive-hover;
+    }
+
+    @variant focus-visible {
+      @apply outline-interactive-hover;
+    }
+
+    @variant focus-within {
+      @apply outline-accent-primary-bold;
+    }
+
+    @variant pressed {
+      @apply outline-interactive-pressed;
+    }
+
+    @variant group-size-medium/combobox-field {
+      @apply min-w-[160px] max-w-[400px];
```

wdyt about using the css props directly here (and line 48)
```suggestion
      min-width: 160px;
      max-width: 400px;
```

---

Yea, that's what I've been doing, just missed these ones. Thanks for the pointer